lxc/main.go

```diff
@@ -22,6 +22,18 @@ import (
 	"github.com/canonical/lxd/shared/version"
 )
 
+var capabilities = []string{
+	"cookiejar",
```

Comment:
Probably need to make this a const in the API package because the OIDC verifier will be looking for the same string in client capabilities before trying to set a cookie.